lib/order.rb

```diff
@@ -0,0 +1,57 @@
+require_relative 'formatter'
+
+class Order
+  include Formatter
+
+  attr_reader :list
+
+  def initialize(time_klass = Time)
+    @time_klass = time_klass
+    @list = Array.new
```

Comment:
Style/EmptyLiteral: Use array literal `[]` instead of `Array.new`. (https://github.com/bbatsov/ruby-style-guide#literal-array-hash)